doc/Development_Documentation/10_E-Commerce_Framework/15_Payment/04_PayPal.md

```diff
@@ -163,9 +163,13 @@ public function handleResponseAction() {
     // ... check order state and redirect user to error page or order success page
     
 } 
+```
 
+6) (Optional) Add `PaymentProviderPayPalSmartButton` ObjectBrick
+If you are updating an existing Pimcore instance (with originated before 6.0.1), you might need to add the 
+`PaymentProviderPayPalSmartButton` to your installation. It can be imported from 
+[this json export file](https://github.com/pimcore/pimcore/tree/master/bundles/EcommerceFrameworkBundle/Resources/install/objectbrick_sources/objectbrick_PaymentProviderPayPalSmartButton_export.json). 

```

Comment:
404 error on this link

Reply:
because this file is part of this PR and not in master yet. should be there once this PR is merged